react/details/.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_SERVER_ENDPOINT=http://localhost:8081/entando/
```

Comment:
Is there any particular reason to have all those env variables? Token + server endpoint (we call it domain in past projects, let's stick to a unique name) should be enough